tests/fuzz/fuzz_rpc/rpc_endpoints.cpp

```diff
@@ -0,0 +1,936 @@
+#include "rpc_endpoints.h"
+#include "initialisation.h"
+#include "rpc/rpc_payment_signature.h"
+#include <cstring>
+#include <map>
+
+// Initialising common objects for calling RPC endpoint functions
+cryptonote::core_rpc_server::connection_context ctx;
```

Comment:
I'm going todo a second pass to make sure this is all legit in here later, wanted todo a general code review first.